build/optprof/UpdateRunSettings.ps1

```diff
@@ -14,11 +14,13 @@ $runsettingsXml = [Xml.XmlDocument](Get-Content $runsettingsPath)
 # https://stackoverflow.com/questions/33813700/empty-xml-node-rendered-as-string-in-powershell
 $testStores = $runsettingsXml.RunSettings.TestConfiguration.SelectSingleNode('TestStores')
 
-$profilingInputsStore = $runsettingsXml.CreateElement('TestStore')
+# https://stackoverflow.com/a/59090765/294804
+$profilingInputsStore = $runsettingsXml.CreateElement('TestStore', $runsettingsXml.DocumentElement.NamespaceURI)
```

Comment:
This simply removes the `xmlns=""` from the element produced.

Reply:
Perhaps add that info in a comment in the script directly.

Reply:
I figured the link comment was more informative.
![image](https://user-images.githubusercontent.com/17788297/157302932-b1aa4368-250b-41c7-a04d-19413c3bf41b.png)